lms/djangoapps/grades/migrations/0013_auto_20170403_1730.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grades', '0012_computegradessetting'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='persistentcoursegrade',
+            name='letter_grade',
+            field=models.CharField(max_length=255, verbose_name='Letter grade for course', blank=True),
```

Comment:
Please verify that this migration doesn't result in a SQL-level migration.
We should also FYI devops when this is to be released to stage/prod.